regnety/models/model.py

```diff
@@ -0,0 +1,74 @@
+import tensorflow as tf
+
+from regnety.regnety.config import get_model_config, ALLOWED_FLOPS
+from regnety.regnety.models.blocks import PreStem, Stem, Stage, Head
+
+from typing import List, Tuple, Union
+
+
+def _get_model_with_config(config,  
+    userdef_input_shape: Union[List, Tuple] = (224,224,3)):
+    """
+    Makes a tf.keras.Sequential model using the given config.
+        
+    """
+    model = tf.keras.models.Sequential()
+    model.add(tf.keras.layers.InputLayer(input_shape=userdef_input_shape))
+    model.add(PreStem())
+    model.add(Stem())
+
+    in_channels = 32  # Output channels from Stem
+
+    for i in range(4):  # 4 stages
+        depth = config.depths[i]
+        out_channels = config.widths[i]
+        group_width = config.group_width
+
+        model.add(Stage(
+            depth,
+            group_width,
+            in_channels,
+            out_channels,
+            stage_num=i
+        ))
+
+        in_channels = out_channels
+
+    model.add(Head(config.num_classes))
+
+    return model
+
+
+def RegNetY(flops: str = "", input_shape: Union[List, Tuple] = None):
```

Comment:
It would be nice to indicate that this returns a tf.keras.Model (with type annotations).